### DISCUSSION TITLE
Build fails on simple clone

### DISCUSSION THREAD

**Discussion by thesidshah** [nextauthjs/next-auth#8704] [Help]
### Environment

System:
    OS: macOS 13.4
    CPU: (10) arm64 Apple M2 Pro
    Memory: 148.41 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.18.0 - ~/.nvm/versions/node/v18.18.0/bin/node
    npm: 9.8.1 - ~/.nvm/versions/node/v18.18.0/bin/npm
  Browsers:
    Chrome: 116.0.5845.187
    Safari: 16.5

### Reproduction URL

https://github.com/nextauthjs/next-auth/

### Describe the issue

The error is:- info Linting and checking validity of types ...Failed to compile.

./auth.ts:3:15
Type error: Module '"next-auth"' has no exported member 'NextAuthOptions'.

  1 | import type { GetServerSidePropsContext, NextApiRequest, NextApiResponse } from "next"
  2 | 
> 3 | import type { NextAuthOptions as NextAuthConfig } from "next-auth" // eslint-disable-line
    |               ^
  4 | import { getServerSession } from "next-auth" // eslint-disable-line
  5 | 
  6 | import Apple from "next-auth/providers/apple"

### How to reproduce

The basic repo itself is not built -  clone the repo and simply delete /app/api or /pages/api [to keep just one of the routing mechanisms] and then simply run npm run build

### Expected behavior

The repo should at least build!

**Comment by thesidshah** [comment]
Any progress guys?!

**Reply by ewzxyh** [reply]
im with the same error
